.github/workflows/dev.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         if: steps.cache-deps.outputs.cache-hit != 'true'
 
       - name: Audit 🛡️
-        run: yarn audit
+        run: yarn audit --groups dependencies
```

Comment:
we should check the devDependencies too here !

Reply:
Sure. 
In my mind, this is a temporary deactivation (while the Storybook team fixes their issues).

But, I will see if we can check the dev dependencies to throw a warning rather than an error.